src/manager.js

```diff
@@ -554,20 +638,22 @@ AdManager.prototype.loadAds = function(element, updateCorrelator, useScopedSelec
      * See Docs here https://developers.google.com/doubleclick-gpt/reference#googletagslot
     */
 
-      if (slot && typeof slot.getSizes == 'function') {
-        gptSlotSizes = slot.getSizes();
-        activeSizes = this.adSlotSizes(gptSlotSizes);
-      } else {
-        adUnitSizes = this.adUnitSizes(adUnitConfig.sizes)[1];
-        activeSizes = adUnitSizes;
-      }
+      activeSizes = this.adUnitSizes(adUnitConfig.sizes);
 
       if (adUnitConfig.amazonEnabled && activeSizes && activeSizes.length) {
         this.fetchAmazonBids(thisEl.id, activeSizes, adUnitConfig.slotName);
       }
     }
 
-    if (typeof window.headertag === 'undefined' || window.headertag.apiReady !== true) {
+    if (this.options.pbjsEnabled && adUnitConfig.pbjsEnabled && !slot.eagerLoad && !adUnitConfig.outOfPage) {
```

Comment:
All of the current slots are set to `eagerLoad: true`, so wouldn't this check prevent `addUnitToPrebid` from being called at all for all those slots?